ocdaction/ocdaction/templates/challenge/challenge_list_archived.html

```diff
@@ -4,17 +4,47 @@
 
 {% block main %}
     <section class="container">
-        <h2>My Archive</h2>
-        {% if challenges %}
-            {% for challenge in challenges %}
-                <ul class="col-xs-6 col-xs-offset-3">
-                    <li class="row">
-                        <a href="{% url 'challenge' challenge.id %}"><h3>{{ challenge.challenge_name }}</h3></a>
-                    </li>
-                </ul>
-            {% endfor %}
+        <h1>My Challenge Archive</h1>
+        {% if challenges_archived %}
+            <div class="col-lg-12">
+                <div class="ibox float-e-margins">
+                    <div class="ibox-title">
+                        <h3 class="text-align-center">Archived</h3>
+                        <div class="ibox-tools">
+                            <a class="close-link">
+                                <i class="fa fa-times"></i>
+                            </a>
+                        </div>
+                    </div>
+                    <div class="ibox-content">
+                        <table class="table table-striped">
+                            <thead>
+                            <tr>
+                                <th>Level</th>
+                                <th>Challenge</th>
+                                <th></th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                                {% for challenge in challenges_archived %}
+                                    <tr>
+                                        <td class="text-align-left ibox-content"></td>
+                                        <td class="text-align-left ibox-content"><a href="{% url 'challenge' challenge.id %}">{{ challenge.challenge_name }}</a></td>
+                                        <td class="text-align-right ibox-content"><a class="" href="{% url 'challenge-score-form' challenge.id anxiety_score_card.id %}">Continue</a></td>
+                                    </tr>
+                                {% endfor %}
+                            </tbody>
+                        </table>
+                    </div>
+                </div>
+            </div>
         {% else %}
-            <h2>You have no archived challenges yet</h2>
+    
+            <div class="custom-panel background--primary text-align-center col-xs-12">
```

Comment:
I can't actually see it rendered yet, but from the code I reckon it's the green colour? (`background--primary`). In Archive list we use beige-ish one: https://app.zeplin.io/project/59dbd75b74d032e1682a78fc/screen/59dbd84774d032e1682a7ee0

But actually an Archive list it's not the part of this ticket changes so you don't have to implement this :)